do.sh

```diff
@@ -301,12 +325,12 @@ function mine_data() {
     grep ovn-installed ${logs} | cut -d ':' -f 2- | tr '|' ' ' \
         | cut -d ' ' -f 1,7 | tr 'T' ' ' | sort > mined-data/ovn-installed.log
 
+    source ${rundir}/${ovn_heater_venv}/bin/activate
     python3 ${topdir}/utils/latency.py "$(date +%z)" \
```

Comment:
The `date +%z` here takes the time zone of the orchestrator, but the value will be used to offset the time taken from the log on the tester node.  This results in very large negative latency values in the mined data.  We should either:

- Get the time zone from the tester container
- Synchronize time zones between the orchestrator and the tester container
- Make ovn-tester to use UTC in logs and remove the time adjustments from the latency calculations.

Reply:
Alternate solution number 4: Only run ovn-tester in the UTC timezone :)

Reply:
I resolved this by removing timedelta logic from `latency.py`.